src/test/scala/uk/gov/hmrc/cache/repository/CacheRepositorySpec.scala

```diff
@@ -36,10 +37,13 @@ class CacheRepositorySpec extends WordSpecLike with Matchers with MongoSpecSuppo
   def await[A](future: Future[A])(implicit timeout: Duration) = Await.result(future, timeout)
 
 
-  private val expireAfter28DaysInSeconds = 60 * 60 * 24 * 7 * 4
+  private val expireAfter28Days = TimeToLive(28.days)
 
-  private def repo(name: String, expiresAfter: Long = expireAfter28DaysInSeconds) = new CacheMongoRepository(name, expiresAfter) {
-    await(super.ensureIndexes)
+  private def repo(name: String, expiresAfter: TimeToLive = expireAfter28Days) = {
+    val repo = new CacheMongoRepository(name, expiresAfter)
```

Comment:
Should we write test in terms of `CacheRepositoryFactory` instead?

Reply:
Factory just hides this details. But using factory would be nice thing.